tests/test_callprofiler.py

```diff
@@ -380,4 +380,47 @@ def foo():
         modmesh.call_profiler.reset()
         foo()
 
+    def test_status(self):
+        modmesh.wrapper_profiler_status.disable()
+        self.assertFalse(modmesh.wrapper_profiler_status.enabled)
+
+        modmesh.call_profiler.reset()
```

Comment:
The original part is to reset TimeRegistry. To migrate into CallProfiler tests, I reset CallProfiler.

```diff
- modmesh.time_registry.clear()
+ modmesh.call_profiler.reset()
```